scripts/exglobal_archive_tars.py

```diff
@@ -54,18 +54,24 @@ def main():
         if key.startswith(("COM_", "COMIN_", "COMOUT_")):
             archive_dict[key] = archive.task_config.get(key)
 
+    pwd = os.getcwd()
     with chdir(config.ROTDIR):
+        logger.debug(f"Changed working directory to {config.ROTDIR}")
 
         # Determine which archives to create
         atardir_sets = archive.configure_tars(archive_dict)
 
         # Create the backup tarballs and store in ATARDIR
         for atardir_set in atardir_sets:
+            logger.debug(f"Processing archive set: {atardir_set['name']}")
             archive.execute_backup_dataset(atardir_set)
 
         # Clean up any temporary files
+        logger.debug("Cleaning up temporary files and directories")
         archive.clean()
 
+    logger.info(f"Returned to working directory {pwd}")
```

Comment:
```suggestion
    logger.info(f"Returned to working directory {os.getcwd()}")
```
You don't know it returned to PWD, unless you call `pwd`.  The `info` will simply write what was stored in the variable `pwd`

Reply:
Thank for the suggestion!